techdocs/docs/using-the-package/troubleshooting.md

```diff
@@ -0,0 +1,39 @@
+# Troubleshooting
+
+Sometimes issues may arise when using the packages for the first time, or when updating to a new version. Use this guide to help troubleshoot the problem.
+
+## Initial Setup Checklist
+
+1. Ensure you follow the Quick Start Guides closely for each package.
+
+  - [SSO React - Quick Start Guide](../getting-started/quick-start-guide.md)
+  - [SSO Express - Quick Start Guide](https://developer.gov.bc.ca/docs/default/component/citz-imb-sso-express-npm-package/getting-started/quick-start-guide/)
+
+2. Ensure environment variables are properly configured in the container running your `citz-imb-sso-express` package. Keep in mind that `SSO_CLIENT_SECRET` and `SSO_AUTH_SERVER_URL` are different between each SSO environment (Development, Test, Production).
+
+3. Ensure your SSO Integration is set up properly. This includes the `Usecase` being set to `Browser Login and Service Account`, and the `Redirect URIs` being properly configured for each SSO environment (Development, Test, Production). For more information, visit [SSO Integration Settings](../getting-started/sso-integration-settings.md).
+
+<br />
+
+## Diagnosing the Problem
+
+If an issue persists after following all the proper setup guides, follow these steps to help find the problem.
+
+1. In the `citz-imb-sso-express` package, you can get more detailed console logs by setting `DEBUG` and `VERBOSE_DEBUG` environment variables to `true`. These logs may help you spot an issue such as a null or undefined value.
+
+2. For your frontend you can check the Application cookies and Network tab in your browser's dev tools to help diagnose the problem.
+
+  - If you see an undefined `refresh_token` cookie, there is a problem.
```

Comment:
"there is a problem" - what is it?  How do I fix it?

Reply:
Its not entirely clear what the problem is but I can add a bit more of a description.